python/rmm/_lib/memory_resource.pyx

```diff
@@ -349,6 +349,13 @@ cpdef _set_default_resource(MemoryResource mr):
     set_default_resource(_mr.c_obj)
 
 
+cpdef get_default_resource_type():
+    """
+    Get the default memory resource type used for RMM device allocations.
+    """
+    return type(_mr)
+
+
 cpdef is_initialized():
     global _mr
     return _mr.c_obj.get() is not NULL
```

Comment:
This is always true, right? It is in the C++.

Reply:
I don't know, but this isn't from this changeset.

cc @shwina

Reply:
In general someone would need to do something very unsafe / shoot themselves in the foot for this to return false, like calling `_mr.c_obj.release()` on it. I don't know why we kept this function but I believe we did so to reduce API churn for the time being.